tests/contest/contest/src/tests/process/process_test.rs

```diff
@@ -0,0 +1,54 @@
+use std::fs;
+use std::os::unix::fs::PermissionsExt;
+
+use anyhow::{bail, Context, Ok, Result};
+use oci_spec::runtime::{ProcessBuilder, Spec, SpecBuilder};
+use test_framework::{test_result, Test, TestGroup, TestResult};
+
+use crate::utils::test_inside_container;
+
+fn create_spec() -> Result<Spec> {
+    let mut process = ProcessBuilder::default()
+        .args(vec!["runtimetest".to_string(), "process".to_string()])
+        .cwd("/test")
+        .build()
+        .expect("error in creating process config");
+    let mut env = process.env().clone().unwrap();
+    env.push("testa=valuea".to_string());
+    env.push("testb=123".to_string());
+    process.set_env(Some(env));
+
+    let spec = SpecBuilder::default()
+        .process(process)
+        .build()
+        .context("failed to build spec")?;
+
+    Ok(spec)
+}
+
+fn process_test() -> TestResult {
+    let spec = test_result!(create_spec());
+
+    test_inside_container(spec, &|bundle| {
+        match fs::create_dir(bundle.join("test")) {
+            Result::Ok(_) => { /*This is expected*/ }
+            Err(e) => {
+                bail!(e)
+            }
+        }
+        let metadata = fs::metadata(bundle.join("test"))?;
+        let mut permissions = metadata.permissions();
+        permissions.set_mode(0o700);
```

Comment:
Hey, what is the reason for this code? 
1. I don't think this will actually set the mode of the dir, as we are only updating the in-memory struct perms. We will need another syscall to actually set the mode, if you are trying to copy the behavior from go test
2. Also I don't get the intention behind this. We are not checking the perms in the runtimetest code, and if I'm correct then test is passing without this actually being set. So either we should test this in the runtime test code, or just remove these lines, as they essentially are no-op.

Reply:
Sorry, I misunderstood and thought it was necessary after reading the runtimes-tools code.

https://github.com/opencontainers/runtime-tools/blob/master/validation/process/process.go#L20-L21

I delete code.
https://github.com/youki-dev/youki/pull/2968/commits/3099f7b774a60ef1eba4b7b68f07a88d5191967e